scripts/SANS/sans/gui_logic/presenter/add_runs_presenter.py

```diff
@@ -10,7 +10,7 @@
 
 from sans.common.enums import SANSInstrument
 from sans.gui_logic.gui_common import GENERIC_SETTINGS, load_property, set_setting
-from sans.gui_logic.models.run_selection import has_any_event_data
+#from sans.gui_logic.models.run_selection import has_any_event_data
```

Comment:
Stray comment

Reply:
That's the import required for disabling the run summation settings unless they contain event data. I've left the import and old code in to make it easier to revert, but should I just remove it completely?

Reply:
I think I would prefer removing it completely as otherwise if the scientists like the current implementation there is a chance we never get around to removing it later.

Reply:
That's pushed